### DISCUSSION TITLE
[Feature Request] Accessibility settings - color blind modes, color grading, contrast, outlines, font changing etc.

### DISCUSSION THREAD

**Discussion by Pinkyyyy** [FakeFishGames/Barotrauma#8712] [Ideas and suggestions]
Hi,

while trying to introduce a new friend to the game we've hit an immediate road block.

They can't see the color the game is toned with well. That sea foam green is really hard for them to see, i thought there might be ui settings or a way to change this there's not. This game doesn't seem to have good accessibility options. No way to change colors, contrast or change font. Another suggestion would be to add an option to have outlines of intractable objects always on, or increase the contrast of said objects, make doors more apparent etc.

They got pretty overwhelmed by the UI, the font choice wasn't helping and no way to change the colors really turned them off from the game. Would be great to see more flexibility in controlling the UI size for example make just the boxes with font bigger. More readable font options like open dyslexic. Color blind options, and options to change contrast of interactive items or a way to tell things that are interactive without having to hover over them, and a way to make doors more readable.

Also the UI for multiplayer lobbies could be simplified so only the host sees all the extra details while the crew only sees the Submarine, Class and Chat, decreasing the overwhelming information on screen that is not needed,  this could also be a setting.

**Comment by cshingle** [comment]
What exactly did they have issues with? 
I am color blind and I have less issues with this game than most as the devs don't change the color of items to convey information like so may games. I can't even think of a single item where the color changes for anything other than for artistic reasons.

**Reply by Pinkyyyy** [reply]
I mean there are different types of color blindness, but also i think the issue is just with the readability of the game, you dont know whats intractable whats not, you have to hover over things to figure it out, that bit confuses even me. So i think having more options to improve the readability of the game is a good idea regardless.
